### PR TITLE
Fix #create_table. Run with mysql_adapter as it's not an alter statement

### PR DESCRIPTION
**Pull request by sauloperez**

### CODE REVIEW

File: spec/integration_spec.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require 'byebug'
```
Comment (enricostano) [inline spec/integration_spec.rb]: still needed? maybe require it as dev dependency

Comment (sauloperez) [inline spec/integration_spec.rb]: Ups, I forgot to remove it. It's already specified in the gemspec it just needs to be autorequired, as done in https://github.com/redbooth/percona_migrator/commit/9afc111da6126e8b430c7405b70b636006c15e9f already.

**Comment by mrcasals** [comment]
:speech_balloon: & :+1:

File: spec/integration_spec.rb (hunk 1 above)
Comment (enricostano) [inline spec/integration_spec.rb]: :ok_hand:

**Comment by enricostano** [comment]
:+1:

**Comment by jrom** [comment]
:+1:

**Pull request merged**